cypress/e2e/tests/migration/applicationinventory/applications/import.test.ts

```diff
@@ -78,7 +79,8 @@ describe(["@tier3"], "Application import operations", () => {
         imports.verifyImportErrorMsg(errorMsgs);
     });
 
-    it("Applications import for non existing tags and BS", function () {
+    it("Bug MTA-4257: 1)Applications import for non existing tags and BS \
+        2)Verify assigned BS for imported application if BS was created through previous import", function () {
```

Comment:
Why are we testing 2 things in the same test case instead of having two different `it`?

Reply:
Thanks @abrugaro.  These are long running tests, so I decided to combine the two to save some time . I've combined tests in the same `it` block  for Assessment tests in the past . Test # 2 requires a few steps from test # 1. 

One disadvantage to combining tests is that it does introduce some dependency.